azavea/docker-openjdk-gdal
# Add OpenJDK11 to build matrix

## Overview

- Add OpenJDK 11 to build matrix
- Pass the JDK home path to GDAL (https://github.com/docker-library/openjdk/commit/3eb0351b208d739fac35345c85e3c6237c2114ec)
- Remove `alpine`-variant images from build matrix (https://github.com/docker-library/openjdk/issues/272)

Connects https://github.com/azavea/raster-foundry-platform/issues/743

## Testing Instructions

See Travis CI build: https://travis-ci.org/azavea/docker-openjdk-gdal/builds/544795557
